### PR TITLE
JENKINS-56707 improved testcoverage of "LinesChartModel" to 100%

### PR DESCRIPTION
**Pull request by LorMun**
Improved the testcoverage for "LinesChartModel".

### CODE REVIEW

**Comment by LorMun** [comment]
@apolovyi

File: src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java [hunk 1]
```diff
@@ -9,15 +10,76 @@
...
  */
+
 class LinesChartModelTest {
+
     private static final String COLOR = "#fff";
 
+    @Test
+    void testEmptyCtor() {
+        LinesChartModel model = new LinesChartModel();
+        assertThat(model.getId().compareTo(""));
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java]:
```
assertThat(model.getId()).isEmpty();
```

File: src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java [hunk 2]
```diff
@@ -9,15 +10,76 @@
...
+        assertThat(model.getSeries().size() == 0);
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java]: Same here:
```
assertThat(model.getSeries()).hasSize(0);
```
or
```
assertThat(model.getSeries()).isEmpty();
```

File: src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java [hunk 3]
```diff
@@ -9,15 +10,76 @@
...
+                + "  \"id\": \"spotbugs\",\n"
+                + "  \"series\": []\n"
+                + "}"));
+    }
+
+    @Test
+    void testSize() {
+        LinesChartModel model = new LinesChartModel("spotbugs");
+        model.addXAxisLabels(Arrays.asList("1", "2", "3"));
+        assertThat(model.size() == 3);
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java]: Check the getter `xAxisLabels` as well

File: src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java [hunk 4]
```diff
@@ -9,15 +10,76 @@
...
+    @Test
+    void testGetXAxisLabels() {
+        LinesChartModel modelForSingleXAxisLabelTest = new LinesChartModel("spotbugs");
+        modelForSingleXAxisLabelTest.addXAxisLabel("a");
+        assertThat(modelForSingleXAxisLabelTest.getXAxisLabels().equals(Arrays.asList("a")));
+
+        LinesChartModel modelForXAxisListLabelTest = new LinesChartModel("spotbugs");
+        modelForXAxisListLabelTest.addXAxisLabels(Arrays.asList("a", "b", "c"));
+        assertThat(modelForXAxisListLabelTest.getXAxisLabels().equals(Arrays.asList("a", "b", "c")));
+    }
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/charts/LinesChartModelTest.java]: Add a test that checks that multiple invocations of the add methods work correctly.

**Comment by LorMun** [comment]
@uhafner discussions should be resolved

**Comment by uhafner** [comment]
Did you forget to push? There is no new commit?

**Pull request closed without merging**